lib/spack/spack/environment/environment.py

```diff
@@ -1035,7 +1035,7 @@ def is_develop(self, spec):
         """Returns true when the spec is built from local sources"""
         return spec.name in self.dev_specs
 
-    def concretize(self, force=False, tests=False):
+    def concretize(self, force=False, tests=False, reuse=False):
```

Comment:
TBD 0.17.1

Reply:
@becker33 In the future, it would probably be better to read it outside and pass it as an argument:
```python
def concretize(self, conf=None):
    ...
```
to decouple calling concretize from having a configuration available (that would simplify mockery for unit-tests for instance). 

For why we have just command line arguments at this point there are two comments https://github.com/spack/spack/pull/25310#issuecomment-960093788 and https://github.com/spack/spack/pull/25310#issuecomment-960102546. Pointing this out since these comments on GitHub are hidden by default for me, so not clear if you disagree with that.